src/components/file-upload/FileUpload.tsx

```diff
@@ -20,8 +20,8 @@ export interface FileUploadProps {
   error?: boolean;
 
   /** Display file info if uploaded file property has data */
-  uploadedFile?: UploadedFileProps;
-  onRemove: () => void;
+  uploadedFile?: UploadedFileProps | null;
+  onRemove?: (() => void) | null;
```

Comment:
Why allow null as well as undefined?

Reply:
The reason for that is if you don't specify it or set it to null, it will not do any action. Does that answer your question? 🤔

Reply:
I guess my question is why would someone set it to null as opposed to just not specifying.

Reply:
Yah, I got your point. I just have a use case that I need to conditionally trigger the action/object. And I don't love to have a falsy ternary with `undefined`. But I think it's good to have either one or the other. Thanks